src/net/mod.rs

```diff
@@ -922,6 +922,7 @@ pub enum HttpRequestType {
     GetTransferCost(HttpRequestMetadata),
     GetContractSrc(HttpRequestMetadata, StacksAddress, ContractName, bool),
     GetContractABI(HttpRequestMetadata, StacksAddress, ContractName),
+    OptionsPreflight(HttpRequestMetadata),
```

Comment:
If we intend to support wildcard paths, then the path should be captured in this type variant as well (so it can be used to reconstruct the request path).

Reply:
Added the path in this type :+1: